app/screens/ChannelsScreen.tsx

```diff
@@ -86,7 +86,12 @@ export const ChannelsScreen: FC<ChannelsScreenProps> = observer(function Channel
     }
 
     async function initChannels(prev) {
-      const res = await channelManager.listChannels(true)
+      let res = await channelManager.listChannels(true)
+
+      // filter
+      res = res.filter((el) => el.name && isImage(el.picture) && el.is_private)
```

Comment:
what if there is no picture?   or no name?  or it's a public channel?   why are we removing this from the system?

Reply:
I've updated it, just get only public channel then merge both array and remove duplicate 
```
// remove private channel
res = res.filter((el) => !el.is_private)

// merge array, remove dup
const final = [...new Map([...prev, ...res].map((el) => [el.id, el])).values()]
```